From 4d9c0e9573269f53a8f833539b6d56dd6936432b Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Thu, 13 Feb 2014 17:46:03 -0500
Subject: [PATCH 11/14] Use os-svc-restart for cinder-api

We need to use os-svc-restart here to restart the cinder-api and
cinder-scheduler services so that map-services is used.

Change-Id: Ib6d3778d73bfcebe05775c86192e005e79f5c128
---
 elements/cinder-api/os-refresh-config/post-configure.d/73-cinder-api | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/elements/cinder-api/os-refresh-config/post-configure.d/73-cinder-api b/elements/cinder-api/os-refresh-config/post-configure.d/73-cinder-api
index 7d23c9c..6155596 100755
--- a/elements/cinder-api/os-refresh-config/post-configure.d/73-cinder-api
+++ b/elements/cinder-api/os-refresh-config/post-configure.d/73-cinder-api
@@ -5,5 +5,5 @@ cinder-manage db sync
 
 os-svc-enable -n cinder-api
 os-svc-enable -n cinder-scheduler
-service cinder-api restart
-service cinder-scheduler restart
+os-svc-restart -n cinder-api
+os-svc-restart -n cinder-scheduler
-- 
1.8.5.3

